tests/simulation/translator/utils/xv15_bet_disk_helper.py

```diff
@@ -333,10 +333,10 @@ def _createBETTwistsAndChords(pitch_in_degree):
     chords = []
     with imperial_unit_system:
         for radius, twist in zip(radial_loc_for_twist, radial_twists_curr):
-            betDiskTwist = BETDiskTwist(radius=radius, twist=twist)
+            betDiskTwist = BETDiskTwist(radius=radius * u.inch, twist=twist * u.deg)
```

Comment:
So without u.inch this does not work? I thought the unit system is going to auto infer the unit?